### PR TITLE
Sync with master with latest updates

### PR DESCRIPTION
**Pull request by abhi18av**
@Aryan-sharma11 , I have created this pull request to keep your work visible.

### CODE REVIEW

File: AddSevenPS.sln.DotSettings.user [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+	<s:String x:Key="/Default/Environment/AssemblyExplorer/XmlDocument/@EntryValue">&lt;AssemblyExplorer&gt;&#xD;
+  &lt;Assembly Path="C:\Users\Aryan Sharma\RiderProjects\AddSevenPS\AddSevenPS\bin\Debug\AddSevenPS.dll" /&gt;&#xD;
+&lt;/AssemblyExplorer&gt;</s:String></wpf:ResourceDictionary>
+=======
+	<s:String x:Key="/Default/Environment/UnitTesting/UnitTestSessionStore/Sessions/=fcd77302_002D8301_002D4c7b_002D82e7_002Dfc99bf0260c9/@EntryIndexedValue">&lt;SessionState ContinuousTestingMode="0" IsActive="True" Name=" AddSeven.addSeven -&amp;gt; &amp;quot; 1 =&amp;gt; 8&amp;quot; " xmlns="urn:schemas-jetbrains-com:jetbrains-ut-session"&gt;&#xD;
+  &lt;TestAncestor&gt;&#xD;
+    &lt;TestId&gt;xUnit::49984BB5-D987-4043-A6B6-4C09CCD60730::net6.0::AddSevenPS.LibraryTest&lt;/TestId&gt;&#xD;
+  &lt;/TestAncestor&gt;&#xD;
+&lt;/SessionState&gt;</s:String></wpf:ResourceDictionary>
+>>>>>>> origin/develop
```
Comment (abhi18av) [inline AddSevenPS.sln.DotSettings.user:12]: @Aryan-sharma11 , this indicates an unclean merge process. 🔴

File: AddSevenPS/Library.fs [hunk 2]
```diff
@@ -1,4 +1,37 @@
...
+open System.Management.Automation
+
+open System.Management.Automation.Runspaces
+
+module AddSeven =
+    let addSeven num = num + 7
+     
+    // For testing purpose
+    //  Testing list of integers and then returning the list with solutions  
+    let add7 numList =
```
Comment (abhi18av) [inline AddSevenPS/Library.fs:12]: For this, I like the idea but the naming convention doesn't convey that this function `add7` can take a list.

As per the [`dotNET` naming guidelines](https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/component-design-guidelines#naming-conventions) `addSevenToList` makes more sense. 

Please refactor :) 


`numList` is good

**Review by abhi18av — COMMENTED** [review]
Added comments for naming convention

File: .gitignore [hunk 3]
```diff
@@ -2,4 +2,5 @@ bin/
 obj/
 /packages/
 riderModule.iml
-/_ReSharper.Caches/
\ No newline at end of file
+/_ReSharper.Caches/
+.idea
```
Comment (abhi18av) [inline .gitignore:6]: I think that you should also add `.vs` over here, that folder is unnecessary in the codebase.
https://github.com/Aryan-sharma11/AddSevenPS/tree/develop/.vs/AddSevenPS/v17


You'll need to update the gitignore and then forcefully commit the removal of `.vs` folder.

File: AddSevenPS.sln.DotSettings.user (hunk 1 above)
Comment (Aryan-sharma11) [reply AddSevenPS.sln.DotSettings.user:12]: > @Aryan-sharma11 , this indicates an unclean merge process. 🔴

Is this file necessary or should I add this too in .gitignore ??

File: AddSevenPS/Library.fs (hunk 2 above)
Comment (Aryan-sharma11) [reply AddSevenPS/Library.fs]: Done

File: .gitignore (hunk 3 above)
Comment (Aryan-sharma11) [reply .gitignore:6]: I resolved this problem in commit 676f1bcfb910d79c27c7ea4ef9a745a8c59e6327

File: AddSevenPS.sln.DotSettings.user (hunk 1 above)
Comment (Aryan-sharma11) [reply AddSevenPS.sln.DotSettings.user]: Yes this also resolved in commit 676f1bcfb910d79c27c7ea4ef9a745a8c59e6327

**Pull request merged**